src/contexts/StateContextProvider.tsx

```
@@ -458,18 +443,33 @@ const StateContextProvider = ({children}: StateContextProviderProps) => {
+            // Avoid redundant loadPageByCursor
+            return;
+        }
+
         setUiState(UI_STATE.FAST_LOADING);
-        loadPageByCursor(mainWorkerRef.current, cursor);
-    }, [loadFile]);
+        loadPageByCursor(mainWorkerRef.current, cursor, isPrettified ?? false);
+    }, [loadFile, isPrettified]);
 
-    const filterLogs = useCallback((filter: LogLevelFilter) => {
+    const filterLogs = useCallback((filter: LogLevelFilter, isPrettified: boolean) => {
```

ESLint

[error] 'isPrettified' is already declared in the upper scope on line 231 column 22.

(@typescript-eslint/no-shadow)